index.js

```diff
@@ -109,31 +98,25 @@ controller.hears(['hello'], ['message_received', 'direct_message', 'direct_menti
   bot.startConversation(message, askName)
 })
 
-let getUsers = function (bot) {
-  bot.api.users.list({}, (err, response) => {
-    if (err) return err
-    // console.log(response.members[0])
-    response.members.forEach((user) => {
+function getUsers (callback) {
+  web.users.list(token, (err, response) => {
+    if (!err) {
+      response.members.forEach((user) => {
       // need to get id for each user and pass it into the save below
-      controller.storage.users.save(user, function (err) {
-        if (err) console.log(err)
+        controller.storage.users.save(user, function (err) {
+          if (err) console.log(err + ' a')
+        })
       })
-    })
-    return response
+      callback(null, response)
+    } else callback(err + ' b')
   })
 }
 
 function getProjects (callback) {
   request('http://jenny-production-ecs-1811095239.eu-west-1.elb.amazonaws.com/api/projects/2017', (error, response, body) => {
```

Comment:
This function should probably take a year parameter.

The endpoint should probably go into the`config.js`. Maybe as `alphaGatewayUrl` and then the code would look like:

```js
function getProjects(year) {
  request(config.alphaGatewayUrl + '/api/projects/' + year, (error, response, body) => {
    // stuff
  })
}
```